File: client-java/controller/src/main/java/org/evomaster/client/java/controller/mongo/operations/LessThanOperation.java
package org.evomaster.client.java.controller.mongo.operations;

/**
 * Represent $lt operation.
 * Selects the documents where the value of the field is less than to a specified value
 */
public class LessThanOperation<V> extends ComparisonOperation<V>{
    public LessThanOperation(String fieldName, V value) {
        super(fieldName, value);
    }
}

File: client-java/controller/src/main/java/org/evomaster/client/java/controller/mongo/operations/LessThanOperation.kt
package org.evomaster.client.java.controller.mongo.operations

/**
 * Represent $lt operation.
 * Selects the documents where the value of the field is less than to a specified value
 */
class LessThanOperation<V>(
    override val fieldName: String,
    override val value: V
): ComparisonOperation<V>(fieldName, value)